crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -1418,7 +1418,10 @@ async fn test_transfer_package() {
         .await
         .unwrap()
         .unwrap();
-    let package_object_ref = authority_state.get_framework_object_ref().await.unwrap();
+    let package_object_ref = authority_state
+        .get_system_package_object_ref()
```

Comment:
nit: "system_package" is used to refer to all these packages collectively, how about `get_sui_system_object_ref`, or `get_sui_system_package_object_ref` if you like?